ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/smartcharging/ClearChargingProfileRequest.java

```diff
@@ -1,13 +1,15 @@
 package eu.chargetime.ocpp.model.smartcharging;
 
-import eu.chargetime.ocpp.PropertyConstraintException;
-import eu.chargetime.ocpp.model.Request;
-import eu.chargetime.ocpp.model.core.ChargingProfilePurposeType;
-import eu.chargetime.ocpp.utilities.MoreObjects;
 import java.util.Objects;
+
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
 
+import eu.chargetime.ocpp.PropertyConstraintException;
+import eu.chargetime.ocpp.model.OcppRequest;
+import eu.chargetime.ocpp.model.core.ChargingProfilePurposeType;
+import eu.chargetime.ocpp.utilities.MoreObjects;
```

Comment:
I assume all those changes happened through auto import. Means lots of unnecessary changes in the files (that will likely cause merge conflicts once we pull in from the fork). Not a show stopper, but next time try to minimize by not applying auto format / import.